data/seeds/cli-example/policies/bridg-campaign-messaging.json

```diff
@@ -0,0 +1,16 @@
+{
+  "id": "bridg_campaign_messaging",
+  "description": "Give bridg's campaign messaging service access to Bridg's resources.",
+  "subjects": [
+    "bridg-campaign-messaging"
+  ],
+  "effect": "allow",
+  "resources": [
+    "rn:bridg:scheduler:schedule",
+    "rn:bridg:<(accounts:[A-z0-9_-]+:brands:[A-z0-9_-]+:):(campaigns|email-engine)(:.+)?>>"
```

Comment:
This regex has some minor syntax issues. The end includes `>>` but only one `>` is needed to close the regex expression. Also `(accounts:[A-z0-9_-]+:brands:[A-z0-9_-]+:):` can become `accounts:[A-z0-9_-]+:brands:[A-z0-9_-]+:` (a capture group is not needed and we don't want to include `:` twice. Thanks for the PR here, otherwise it looks good.